src/ResourceManagement/HDInsight/HDInsight.Tests/ScenarioTests/CreateClusterTests.cs

```diff
@@ -423,5 +423,75 @@ public void TestCreateLinuxSparkClusterWithComponentVersion()
                 Assert.Equal(result.State, AsyncOperationState.Succeeded);
             }
         }
+
+        [Fact]
+        public void TestCreateDefaultFsAzureBlobClusterUsingClusterParameters()
+        {
```

Comment:
It is not possible to convert this to a unit test.
To convert to a unit test, I need to be able to setup a mock of IClusterOperations that actually executes the body of #1 below and verifies the call on #2 below.
When #1 completes executing, it would have created a ClusterCreateParametersExtended object and passed it as an argument to #2. We could then have called Mock.Verify with It.Is<ClusterCreateParametersExtended> on #2.
Unfortunately, I dont think it is possible to setup the mock to fully execute certain methods (as though no mocking was setup) and mock the others.

1. public async Task<ClusterGetResponse> CreateAsync(string resourceGroupName, string clusterName, ClusterCreateParameters clusterCreateParameters, CancellationToken cancellationToken)
2. public async Task<ClusterGetResponse> CreateAsync(string resourceGroupName, string clusterName, ClusterCreateParametersExtended clusterCreateParameters, CancellationToken cancellationToken)